kong/db/strategies/cassandra/init.lua

```diff
@@ -575,9 +561,19 @@ function _mt:insert(entity, options)
       end
 
       insert(args, serialize_arg(field, entity[field_name]))
+      insert(cols, field_name)
+      insert(binds, "?")
```

Comment:
Respecting the `insert(cols); insert(binds); insert(args)` order (as per the `is_partitioned` branch) might be helpful for the reader.